crates/diesel_models/src/payment_attempt.rs

```diff
@@ -81,11 +62,19 @@ pub struct PaymentAttempt {
     pub profile_id: id_type::ProfileId,
     pub organization_id: id_type::OrganizationId,
     pub card_network: Option<String>,
```

Comment:
We should not directly flatten the payment_method data in attempt, remove this field for now.